packages/components/src/internal/components/domainproperties/samples/SampleTypePropertiesPanel.tsx

```diff
@@ -338,6 +358,37 @@ class SampleTypePropertiesPanelImpl extends React.PureComponent<
                         </Col>
                     </Row>
                 )}
+
+                {allowTimepointProperties && !appPropertiesOnly && (
+                    <Row className="margin-top">
+                        <Col xs={2}>
+                            <DomainFieldLabel
+                                label="Auto-Link Data to Study"
+                                helpTipBody={
+                                    <>
+                                        <p>
+                                            Automatically link Sample Type data rows to the specified target study. Only
+                                            rows that include subject and visit/date information will be linked.
+                                        </p>
+                                        <p>
+                                            The user performing the import must have insert permission in the target
+                                            study and the corresponding dataset.
```

Comment:
I'm not entirely sure what ends up happening on the study side for this, but should the end of this sentence say "sample type" instead of "dataset"?

Reply:
Nope, the import does end up in the new 'Sample Type Dataset' that Karl is doing the [dev](https://github.com/LabKey/platform/pull/2099) on 👍